keyboards/zeal60/keymaps/ansi_split_bs_rshift/keymap.c

```diff
@@ -0,0 +1,39 @@
+// ANSI split backspace/right shift layout for Zeal60
+
+#include "zeal60.h"
```

Comment:
You can replace this with `#include QMK_KEYBOARD_H`

Reply:
fixed

Reply:
fixed